<launch>
<param name="ROS_IP" value="128.238.39.183" />
<arg name="odom_topic" value="$(env MAV_NAME)/odom"/>

<node name="perception_server_node" pkg="scene_understanding_pkg" type="perception_server" args="--wait" output="screen" respawn="true" >
<remap from="odom" to="$(arg odom_topic)"/>
<remap from="cloud_in" to="cloud_in"/>
</node>


<!--node name="Hololens_Mesh_Data" pkg="scene_understanding_pkg" type="display_received_meshes_in_rviz" args="wait" output="screen" respawn="true" /-->

 <!-- UNITY COMM PARAMS -->
 <group ns="perception_params">
      <rosparam file="$(find scene_understanding_pkg)/config/kimera_data_params.yaml" />
 </group>
</launch>
